### PR TITLE
Enh ext ahmed body loss

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Description

Closes https://github.com/NVIDIA/modulus-launch/issues/82

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is
up to date with these changes.

### CODE REVIEW

File: examples/cfd/ahmed_body_mgn/train.py [hunk 1]
```diff
@@ -251,3 +259,5 @@ def validation(self):
             logger.info(f"Saved model on rank {dist.rank}")
         start = time.time()
     rank_zero_logger.info("Training completed!")
+
+info("Training completed!")
```
Comment (NickGeneva) [inline examples/cfd/ahmed_body_mgn/train.py:263]: logger.info

**Review by NickGeneva — CHANGES_REQUESTED** [review]

**Review by NickGeneva — APPROVED** [review]

File: examples/cfd/ahmed_body_mgn/train.py (hunk 1 above)
Comment (mnabian) [reply examples/cfd/ahmed_body_mgn/train.py]: Fixed

**Pull request merged**